mainnet/README.md

```diff
@@ -1,2 +1,215 @@
-## Installation of EigenDA using docker
-Coming soon...
\ No newline at end of file
+# Mainnet
+
+> **_INFO:_**
+Please ensure you have reviewed the [current Active Operator Set cap](https://docs.eigenlayer.xyz/operator-guides/avs-installation-and-registration/eigenda-operator-guide#eigenda-churn-approver) and ensure you have sufficient delegated restaked ETH TVL before proceeding.
+<!-- ::: -->
+
+#### Step 1: Complete EigenLayer Operator Registration
+
+- Complete the EigenLayer CLI installation and registration [here](https://docs.eigenlayer.xyz/operator-guides/operator-installation).
+
+#### Step 2: Install Docker
+
+- Install [Docker Engine on Linux](https://docs.docker.com/engine/install/ubuntu/).
+
+#### Step 3: Prepare Local EigenDA files
+
+##### Step 3.1: Clone this repo and execute the following commands:
+
+```
+git clone https://github.com/Layr-Labs/eigenda-operator-setup.git
+cd eigenda-operator-setup/mainnet
+cp .env.example .env
+```
+
+
+##### Step 3.2: Manually update the `.env` file downloaded in the steps above. Modify all sections marked with `TODO` to match your environment.
+
+##### Step 3.3: Create local folders which are required by EigenDA:
+
+```
+mkdir -p $HOME/.eigenlayer/eigenda/mainnet/logs
+mkdir -p $HOME/.eigenlayer/eigenda/mainnet/db
+```
+
+##### Step 3.4 Test the values of your environment variables $USER_HOME, $EIGENLAYER_HOME, $EIGENDA_HOME to ensure all the folders exist as expected. Review the resulting values Correct any folders whose paths are not set as expected.
+
+```
+source .env
+ls $USER_HOME $EIGENLAYER_HOME $EIGENDA_HOME
+```
+
+#### Step 4: Operator Networking Security Setup
```

Comment:
I realized we need to put another port in here for operators to open.  in addition to dispersal and retrieval - we should ask them to open `NODE_API_PORT` to internet so that we can get some basic node data from them. I know we didn't ask on holesky but we should add it. So we can say something like

Node API port: 
In order to get some basic data please also open `NODE_API_PORT` in [.env](./.env.example#L16) to internet. The data which we will get from that API is mentioned in the [Node API Spec](https://docs.eigenlayer.xyz/eigenlayer/avs-guides/spec/api/)

Reply:
maybe add this in holesky too with this PR?

Reply:
Is it mandatory for operators to open this port to Internet ingress?

Reply:
it's not mandatory for eigenda working but we want them to open it - so that we can get some more data in terms of what versions they are running and eventually more details on all operator set.

Reply:
@shrimalmadhur does opening that port create any security issues for Operators? Or perhaps "not" because its only a read-only API?

Does the following revised text work for you?
> Node API port:
In order to consolidate Operator metrics to measure the health of the network, please also open NODE_API_PORT in [.env](https://github.com/Layr-Labs/eigenda-operator-setup/pull/.env.example#L16) to the internet if possible. Please see [Node API Spec](https://docs.eigenlayer.xyz/eigenlayer/avs-guides/spec/api/) for more detail on the data made available via this port.

Reply:
yea it's a readonly port. nothing I see as a concern. yea let's just make it optional. the updated text looks real good. thanks

Reply:
once you add this I am good to go with this PR

Reply:
@shrimalmadhur Node API port content has been added to this PR.

Please approve (and Merge on my behalf since I don't have permissions) when ready.